content/blog/2016-05-09-when-to-use-varbit-chunks.md

```diff
@@ -0,0 +1,159 @@
+---
+title: When (not) to use varbit chunks
+created_at: 2016-05-09
```

Comment:
Sure. If we cannot get this out before the post tomorrow, we should probably delay by a day or two. I would really like to get it out soon as I want to point several external and internal people to it. Ideally, we can get it out dated today (May 8), thanks to your speedy feedback. Then it would not overshadow the CNCF post as it would simply appear further down.